modules/openxoutstreamBidAdapter.js

```diff
@@ -0,0 +1,223 @@
+    res: `${screen.width}x${screen.height}x${screen.colorDepth}`,
+    ifr: isInIframe,
+    tz: new Date().getTimezoneOffset(),
+    tws: getViewportDimensions(isInIframe),
+    be: 1,
+    bc: bids[0].params.bc || `${BIDDER_CONFIG}_${BIDDER_VERSION}`,
+    auid: '540141567',
+    dddid: utils._map(bids, bid => bid.transactionId).join(','),
+    openrtb: '%7B%22mimes%22%3A%5B%22video%2Fmp4%22%5D%7D',
+    nocache: new Date().getTime()
+  };
+
+  if (utils.deepAccess(bidderRequest, 'gdprConsent')) {
+    let gdprConsentConfig = bidderRequest.gdprConsent;
+
+    if (gdprConsentConfig.consentString !== undefined) {
+      defaultParams.gdpr_consent = gdprConsentConfig.consentString;
+    }
+
+    if (gdprConsentConfig.gdprApplies !== undefined) {
+      defaultParams.gdpr = gdprConsentConfig.gdprApplies ? 1 : 0;
+    }
+
+    if (config.getConfig('consentManagement.cmpApi') === 'iab') {
+      defaultParams.x_gdpr_f = 1;
+    }
+  }
+
+  return defaultParams;
+}
+
+function buildOXBannerRequest(bids, bidderRequest) {
+  let queryParams = buildCommonQueryParamsFromBids(bids, bidderRequest);
+  queryParams.aus = utils._map(bids, bid => utils.parseSizesInput(bid.sizes).join(',')).join('|');
+
+  if (bids.some(bid => bid.params.doNotTrack)) {
+    queryParams.ns = 1;
+  }
+
+  if (bids.some(bid => bid.params.coppa)) {
+    queryParams.tfcd = 1;
+  }
+
+  let url = `https://${bids[0].params.delDomain}/v/1.0/avjp`
+  return {
+    method: 'GET',
+    url: url,
+    data: queryParams,
+    payload: {'bids': bids}
+  };
+}
+
+function handleVastResponse(response, serverResponse) {
+  const body = response.body
+  let bidResponses = [];
+  if (response !== undefined && body.vastUrl !== '' && body.pub_rev && body.pub_rev > 0) {
+    let vastQueryParams = parse(body.vastUrl).search || {};
+    const openHtmltag = '<html><head></head><body>';
+    const closeHtmlTag = '</body></html>';
+    const sdkScript = createSdkScript().outerHTML;
+    const placementDiv = createPlacementDiv();
+    placementDiv.dataset.pID = PUBLISHER_ID;
+    const placementDivString = placementDiv.outerHTML;
+    const adResponse = getTemplateAdResponse(body.vastUrl, PLACEMENT_ID);
+    const adResponseString = JSON.stringify(adResponse);
+    const ymAdsScript = '<script type="text/javascript"> window.__ymAds =' + adResponseString + '</script>';
+
+    let bidResponse = {};
+    bidResponse.requestId = serverResponse.bids[0].bidId;
+    bidResponse.bidderCode = BIDDER_CODE;
+    bidResponse.netRevenue = NET_REVENUE;
+    bidResponse.currency = CURRENCY;
+    bidResponse.cpm = Number(body.pub_rev) / 1000;
+    bidResponse.creativeId = body.adid;
+    bidResponse.height = body.height;
+    bidResponse.width = body.width;
+    bidResponse.vastUrl = body.vastUrl;
+    bidResponse.ttl = TIME_TO_LIVE;
+    bidResponse.mediaType = BANNER;
+    bidResponse.ad = openHtmltag + placementDivString + ymAdsScript + sdkScript + closeHtmlTag;
+
+    response.ph = vastQueryParams.ph;
```

Comment:
why are we adding/changing parameters in response?

Reply:
because I forgot to delete this. its removed now.